### PR TITLE
Implement get_account_info() to get authenticated user's account info (#556)

### PR DESCRIPTION
**Pull request by apastel**
Addresses #556 

My first PR into the project so please make sure I didn't miss anything or do anything incorrectly!

Here's how it looks in the documentation:

image

### CODE REVIEW

File: tests/mixins/test_library.py [hunk 1]
```diff
@@ -109,3 +111,9 @@ def test_rate_playlist(self, yt_auth):
     def test_subscribe_artists(self, yt_auth):
         yt_auth.subscribe_artists(["UCUDVBtnOQi4c7E8jebpjc9Q"])
         yt_auth.unsubscribe_artists(["UCUDVBtnOQi4c7E8jebpjc9Q"])
+
+    def test_get_account_info(self, config, yt_oauth):
```
Comment (sigma67) [inline tests/mixins/test_library.py:115]: Please test with a non-authenticated account

If it doesn't work, make sure to raise an Exception in the function and expect it here with `pytest.raises`

File: ytmusicapi/navigation.py [hunk 2]
```diff
@@ -78,6 +78,19 @@
...
+    "openPopupAction",
+    "popup",
+    "multiPageMenuRenderer",
+    "header",
+    "activeAccountHeaderRenderer",
+]
+ACCOUNT_RUNS_TEXT = ["runs", 0, "text"]
+ACCOUNT_NAME = [*ACCOUNT_INFO, "accountName", *ACCOUNT_RUNS_TEXT]
+ACCOUNT_CHANNEL_HANDLE = [*ACCOUNT_INFO, "channelHandle", *ACCOUNT_RUNS_TEXT]
+ACCOUNT_PHOTO_URL = [*ACCOUNT_INFO, "accountPhoto", "thumbnails", 0, "url"]
```
Comment (sigma67) [inline ytmusicapi/navigation.py:93]: I'd prefer to put these constants in the `parse_account_info` function for now, they're unlikely to be used anywhere else

File: tests/README.rst [hunk 3]
```diff
@@ -24,6 +24,9 @@ The entry descriptions should be self-explanatory.
 For the headers_raw, you need to indent the overflowing lines with a tab character. For the upload test you need a suitable music file in the test directory.
 Adjust the file to contain appropriate information for your YouTube account and local setup.
 
+The account name and channel handle for test.cfg can be obtained by visiting either YouTube or YouTube Music and
+clicking your profile picture icon in the top right.
```
Comment (sigma67) [inline tests/README.rst:28]:
```suggestion
clicking your profile picture icon in the top right while signed in with your test account.
```

File: tests/README.rst [hunk 4]
```diff
@@ -24,6 +24,9 @@ The entry descriptions should be self-explanatory.
 For the headers_raw, you need to indent the overflowing lines with a tab character. For the upload test you need a suitable music file in the test directory.
 Adjust the file to contain appropriate information for your YouTube account and local setup.
 
+The account name and channel handle for test.cfg can be obtained by visiting either YouTube or YouTube Music and
```
Comment (sigma67) [inline tests/README.rst:27]:
```suggestion
``account_name`` and ``channel_handle`` for test.cfg can be obtained by visiting either YouTube or YouTube Music and
```

**Review by sigma67 — COMMENTED** [review]
Looks really good!

I don't really see a need for extacting the parsing to the parsers/library module, as it's really simple and not likely to be reused anywhere. But feel free to keep as is

File: ytmusicapi/navigation.py (hunk 2 above)
Comment (apastel) [reply ytmusicapi/navigation.py]: I moved them in to `get_account_info` since I took your suggestions and moved the code out of the library parser

**Pull request merged**